src/main/java/frc/robot/RobotContainer.java

```diff
@@ -45,7 +44,9 @@
 
   // The robot's subsystems and commands are defined here...
   private static final Joystick driverJoystick = new Joystick(Constants.portNumber0);
+  private static final Joystick driverJoystick2 = new Joystick(Constants.portNumber1);
```

Comment:
Carry-over from Controller-Layouts branch, which isn't merged yet/has feedback. Instead of trying to do both at once, could this PR just have the test mode controller layout for climb?